common/events.d.ts

```diff
@@ -18,10 +18,51 @@ interface MoveCameraEvent {
     lerp: number;
 }
 
+interface DialogConfirmedEvent {
+    PlayerID: PlayerID;
```

Comment:
All events already contain a PlayerID when they arrive at the server, no need to add it here.